src/index.ts

```diff
@@ -62,6 +63,8 @@ const init: InitCMP = ({ pubData, country }) => {
 		log('cmp', 'initComplete');
 	});
 
+	initVendorDataManager();
```

Comment:
I'll drop it below `resolvedInitialized`, so it doesn't hold anything up hopefully. I don't think it would though, `getConsentFor` is just searching the consent string, and I don't think cookie/localstorage delete calls are very taxing? Could be wrong though!

Reply:
Thanks for clearing up `getConsentFor`. Yep reading and writing cookies is typically fast enough to not worry about but it is a blocking operation which can add up on slower devices and CMP is typically the very first thing that runs on the page. As long as we have metrics we are good to go 👍 🚢

Reply:
yeah makes sense👍 I'll add `requestIdleCallback`